src/eset.c

```diff
@@ -155,6 +156,77 @@ eset_remove(eset_t *eset, edata_t *edata) {
 	    cur_extents_npages - (size >> LG_PAGE), ATOMIC_RELAXED);
 }
 
+edata_t *
+eset_enumerate_alignment_search(eset_t *eset, size_t size, pszind_t bin_ind,
+    size_t alignment) {
+	if (edata_heap_empty(&eset->bins[bin_ind].heap)) {
+		return NULL;
+	}
+
+	void *bfs_queue[ESET_ENUMERATE_NUM];
+	uint16_t front, rear;
+	size_t queue_size, visited_num;
+	edata_t *edata = NULL;
+	edata_heap_enumerate_prepare(&eset->bins[bin_ind].heap, bfs_queue,
```

Comment:
Does the enumerate work if there are unmerged items in the ph aux list?

Reply:
Yes, both `next` and `lchild` are enumerated and unmerged items in the ph aux list are on the `next` chain starting from the root.